packages/esm-form-engine-app/package.json

```diff
@@ -12,7 +12,7 @@
     "debug": "npm run serve",
     "build": "webpack --mode production --color",
     "analyze": "webpack --mode=production --env analyze=true",
-    "lint": "cross-env TIMING=1 eslint src --ext tsx,ts --fix --max-warnings=0",
+    "lint": "cross-env eslint src --ext tsx,ts --fix --max-warnings=0",
```

Comment:
We could drop the `cross-env` here too, since the only reason for that is to allow `TIMING=1`....